recipes/graphviz/yum_requirements.txt

```diff
@@ -0,0 +1 @@
+gtk2-devel
```

Comment:
Do we need this? It looks like we disabled it [above]( https://github.com/conda-forge/staged-recipes/pull/568/files#diff-7da25bc35531c63f94cd50ccc89c8729R21 ).